packages/api-rest/src/types/index.ts

```diff
@@ -64,3 +64,10 @@ export interface apiOptions {
 	endpoints: object;
 	credentials?: object;
 }
+
+export type ApiInfo = {
+	endpoint: string;
+	region?: string;
+	service?: string;
+	custom_header?: () => { [key: string]: string };
```

Comment:
Shouldn't this be plural?

```suggestion
	custom_headers?: () => { [key: string]: string };
```

Reply:
If this name bubbles up all the way to the public API, we might have to live with it in its singular form as to not break consumers.

Reply:
is all the way up :(